runtime/doc/autocmd.txt

```diff
@@ -843,6 +844,21 @@ MenuPopup			Just before showing the popup menu (under the
 					o	Operator-pending
 					i	Insert
 					c	Command line
+MenuPopupChanged 					*MenuPopupChanged*
+				After each time popup menu changed, not fired
+				on popup menu hide, use |CompleteDone| for popup
+				menu hide.
+
+				Sets these |v:event| keys:
+				    completed_item
+				    height
+				    with
+				    row
+				    col
+				    scrollbar
+
+				It is not allowed to change the text |textlock|.
```

Comment:
balloonexpr seems to have the following text. Would be good to having something similar.

>	It is not allowed to change text or jump to another window while
	evaluating 'balloonexpr' |textlock|.

Reply:
But `'balloonexpr'` doesn't exists on neovim, that line is copied from other similar autocmds.

Reply:
I mean we should change the text too.

>It is not allowed to change text or jump to another window while
evaluating 'MenuPopupChanged' |textlock|.

It is a nit ocmment so feel free to ignore.